sdk/core/azure-core/inc/ApiViewSettings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "sourceFilesToProcess": null,
+  "sourceFilesToSkip": [
+    "azure/core/internal/json/json.hpp",
+    "azure/core/internal/json/json_optional.hpp"
```

Comment:
Unlike `json.hpp`, these methods are specific to our repo and written as part of `Azure::Core`. Hencse, it shouldn't be excluded.

Reply:
json.hpp is excluded because the C++ language functionality used by the json package goes well beyond the ability of the tool to process.

In order to actually ship on time, I made the judgement call that it was better to have a tool which works for 99% of the SDK than to have no tool at all.

Reply:
It makes sense to exclude `json.hpp`. I am asking why exclude `json_optional.hpp`?

Is it transitively bringing in `json.hpp` as well, or can it be included standalone?

Reply:
`json_optional.hpp` includes `json.hpp`. I can't include `json_optional.h` without also pulling in `json.hpp`. If I could have comments in json files, I would add that

Reply:
Oh, that's interesting. No other header file in the repo includes `json_optional.h` (which could bring in `json.hpp`). It is only included in source cpp, private header files, or test files.

Makes sense.